app/macos/Podfile.lock

```diff
@@ -155,4 +155,8 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: ce7dbe26c78bfc7ba46736094c1e2d25982870fa
 
+<<<<<<< HEAD
 COCOAPODS: 1.14.2
+=======
+COCOAPODS: 1.14.3
+>>>>>>> main
```

Comment:
pretty sure this deosn't belong here.

Reply:
Yeah it doesn't, thanks for pointing out.